MORK.xcodeproj/project.pbxproj

```diff
@@ -17,7 +17,6 @@
 		52C1BD971AEA7D6C0011A1B9 /* ORKQuestionResult+MORK.m in Sources */ = {isa = PBXBuildFile; fileRef = 52C1BD951AEA7D6C0011A1B9 /* ORKQuestionResult+MORK.m */; };
 		52C1BD981AEA85900011A1B9 /* ORKQuestionResult+MORK.m in Sources */ = {isa = PBXBuildFile; fileRef = 52C1BD951AEA7D6C0011A1B9 /* ORKQuestionResult+MORK.m */; };
 		52EDDD751AE995CF003B0B17 /* MORK.podspec in Resources */ = {isa = PBXBuildFile; fileRef = 52EDDD741AE995CF003B0B17 /* MORK.podspec */; };
-		52EDDD781AE99920003B0B17 /* Podfile in Resources */ = {isa = PBXBuildFile; fileRef = 52EDDD771AE99920003B0B17 /* Podfile */; };
```

Comment:
No need to include the podfile in the actual xcode project itself; it'll get included automatically in the Pods project (included in the MORK project) once `pod install` is run.